config-overrides.js

```diff
@@ -20,6 +20,12 @@ module.exports = {
 		// Expose a dev flag for configuration of environment variables
 		config.plugins.push(new webpack.DefinePlugin({ __DEV__: env !== 'production' }));
 
+		config.ignoreWarnings = config.ignoreWarnings || [];
+		config.ignoreWarnings = [
+			...config.ignoreWarnings,
+			/Failed to parse source map from.*@ahryman40k\/ts-fhir-types.*/,
```

Comment:
@webdevius is this normal?  Not an expert here...

Reply:
No it's not, but the underlying `Ahryman40k/typescript-fhir-types` dependency does not ship its source code (likely a bundling configuration issue there)
so our dev console is spammed with warning messages we can't do much about 😅